docs/source/tutorials/mantid_basic_course/fitting_data/01_fitting_models_to_data.rst

```diff
@@ -101,7 +106,6 @@ C. **Output workspaces** will be created and available via the main Mantid Works
 .. |AddOtherFunctionOption.png| image:: /images/AddOtherFunctionOption.png
    :width: 500px
 .. |ChooseExpDecay.png| image:: /images/ChooseExpDecay.png
-   :width: 150 px
 .. |RunFitOption.png| image:: /images/RunFitOption.png
```

Comment:
With the image put in as a `.. figure` you don't need to define it here.

Reply:
Good point